### PR TITLE
prepping for dockerization.

### PR DESCRIPTION
**Pull request by sneak**

### CODE REVIEW

File: config/defaults.json [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+  "cli_wallet_connection": "",
+  "db": {
+    "dialect": "sqlite",
+    "language": "en",
+    "storage": "/tmp/db.sqlite",
+    "sync": {
+      "force": true
+    }
+  },
+  "db-prod-example": {
```
Comment (valzav) [inline config/defaults.json]: What is the purpose of db-prod-example entry?

File: config/defaults.json [hunk 2]
```diff
@@ -0,0 +1,135 @@
+{
+  "cli_wallet_connection": "",
+  "db": {
```
Comment (valzav) [inline config/defaults.json]: Not sure it will work with sqlite. Did anybody test is with sqlite?

File: config/defaults.json (hunk 1 above)
Comment (sneak) [inline config/defaults.json]: Documentation for people who want to see how it works with mysql. JSON does not allow comments.

File: config/defaults.json (hunk 2 above)
Comment (sneak) [inline config/defaults.json]: I did basic testing. API event logging works. Page view counters work. Haven't tested signups etc.

Comment (valzav) [inline config/defaults.json]: That's nice - sqlite is much easier to install

File: config/defaults.json (hunk 1 above)
Comment (sneak) [inline config/defaults.json]: It ultimately is temporary as the defaults will be set in js soon enough.

File: config/defaults.json (hunk 2 above)
Comment (sneak) [inline config/defaults.json]: Yeah I figure fewer developer deps is a win. I think we should update your raw queries to use orm methods though for max compatibility.

Comment (sneak) [inline config/defaults.json]: (And even if we don't, we should use prepared statements with placeholders instead of string interpolation/concatenation.)

Comment (sneak) [inline config/defaults.json]: I would love to switch to Postgres in prod eventually, it is vastly preferable to mysql.

**Comment by sneak** [comment]
different effort

**Pull request closed without merging**